src/ts/ssh-tcp/services/portForwardingService.ts

```diff
@@ -750,8 +755,10 @@ export class PortForwardingService extends SshService {
 				this.localForwarders.delete(remotePort);
 			});
 
+			this.raiseReportProgress(Progress.CompletedLocalPortForwarding);
 			return localPort;
 		} else if (remotePort !== 0) {
+			this.raiseReportProgress(Progress.CompletedRemotePortForwarding);
```

Comment:
The name of this event might be confusing because the distinction here is not about whether the port being forwarded is local or remote, but whether the client is listening on a local socket for the port or not. (For the latter case the app may still open streams for connections to the port.) In both cases the port is being forwarded _from_ the local side _to_ the remote side. The reverse is also possible but is not currently done with dev tunnels.

Reply:
Will make it CompletedPortForwarding for both. We can make if more specific later if necessary.

Reply:
actually, removed port forwarding and channel opening events for the time being